networks

```diff
@@ -13,6 +13,7 @@ fuse
 gnosis
 gravity
 immutablezkevm
+kaia
```

Comment:
please could you also update config/networks.json (while they are still running in parallel)?